### ISSUE TITLE
Generate imports: Use more python-like preview

### ISSUE THREAD

**Issue by ThiefMaster** [microsoft/pylance-release#1250]
![image](https://user-images.githubusercontent.com/179599/117490421-08df1380-af6f-11eb-8dd9-1fad83bd49c5.png)

I would prefer the items in this dropdown to be shown like this:

```python
from indico.web.flask.app import make_app
from indico.modules.categories.controllers.util make_happening_now_func
```

Ideally with the same syntax highlighting that's used in the editor.

This would make it much more readable than the current way.

**Comment by jakebailey** [comment]
Yep, this is on our TODO list; we made completions more consistent but haven't gone and changed these. Quick fixes are trickier because we can't make the "code" bit monospaced, and the quick fix list contains fixes from things that aren't Pylance as well, so we need to have a balance.

My guess is that we can do something like:

```
Add "from indico.web.flask.app import make_app"
Add "from indico.modules.categories.controllers.util make_happening_now_func"
```

To clearly indicate the action that will be taken.

**Comment by ThiefMaster** [comment]
Lack of monospace sucks :/ Non-monospaced code looks pretty strange. Any idea if I can use custom CSS to make that whole dropdown monospaced? I think I'd prefer that over having nothing in there monospaced...

**Comment by jakebailey** [comment]
Not that I'm aware of, anyway. IIRC that's just the system's default text, or whatever Electron preselects. In any case, swapping the order is an improvement anyway, with or without the monospacing (and other languages like TS behave just fine).

**Comment by heejaechang** [comment]
we can't do coloring either. the only thing we can do is changing the text to reflect what will be inserted. (same as the one in completion)

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.5.4, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202154-26-may-2021
